src/ports/postgres/modules/utilities/sessionize.py_in

```diff
@@ -35,41 +36,83 @@ def sessionize(schema_madlib, source_table, output_table, partition_expr,
         @param source_table: str, Name of the input table/view
         @param output_table: str, Name of the table to store result
         @param partition_expr: str, Expression to partition (group) the input data
-        @param time_stamp: str, Column name with time used for sessionization calculation
+        @param time_stamp: str, The time stamp column name that is used for sessionization calculation
         @param max_time: interval, Delta time between subsequent events to define a session
-
+        @param output_cols: str, list of columns the output table/view must contain (default '*'):
+                        * - all columns in the input table, and a new session ID column
+                        'a,b,c,...' -  a comma separated list of column names/expressions to be projected, along with a new session ID column
```

Comment:
> We will have to decide if this is too hard a constraint to have or not.
> Having this constraint will take away all the messy string parsing stuff
> we currently have implemented though.

Yeah, I didn't realize what the code was doing. I think it would be nice 
to allow the user to choose different output names if they want.

Perhaps a good compromise would be to detect ' AS ' in the string and 
then treat it as a raw select clause. Another option would be to treat 
an array as a list of columns and anything else as a select clause 
(which would also support the * case, I think).